lib/engine/game/g_1868_wy/game.rb

```diff
@@ -33,6 +33,22 @@ class Game < Game::Base
           %w[20b 30b 40o 50o 55y 60y 64 68 72],
           ['', '20b', '30b', '40o', '50o', '55y', '60y'],
         ].freeze
+
+        def init_tiles
+          tiles = super
```

Comment:
super.each do 
end

may work? cuz it returns self

Reply:
ah yep, that works